cmd/internal/cli/actions.go

```diff
@@ -59,26 +60,31 @@ func handleOCI(cmd *cobra.Command, u string) (string, error) {
 		return "", fmt.Errorf("failed to get SHA of %v: %v", u, err)
 	}
 
+	imgabs := ""
 	name := uri.GetName(u)
-	imgabs := cache.OciTempImage(sum, name)
 
-	if exists, err := cache.OciTempExists(sum, name); err != nil {
-		return "", fmt.Errorf("unable to check if %v exists: %v", imgabs, err)
-	} else if !exists {
-		sylog.Infof("Converting OCI blobs to SIF format")
+	if DisableCacheActions {
+		file, err := ioutil.TempDir(tmpDir, "single_use_cache_")
```

Comment:
Theres probably a better name for `single_use_cache_`?...

Reply:
i would use `-` instead of `_` since i think most of the tmp paths we create use those. Maybe call it `tmp-cache-`?